linera-base/src/graphql.rs

```diff
@@ -44,8 +44,8 @@ macro_rules! bcs_scalar {
 
             fn to_value(&self) -> $crate::async_graphql::Value {
                 let ::std::result::Result::Ok(bytes) = $crate::bcs::to_bytes(self) else {
-                                                    return $crate::async_graphql::Value::Null;
-                                                };
+                    return $crate::async_graphql::Value::Null;
+                };
```

Comment:
Not sure why, but there were some random formatting changes when I ran `cargo +nightly fmt`

Reply:
I had something similar and I would recommend doing `cargo fmt`.

Reply:
Huh, I didn't know the nightly version would have such a big difference. Our contributing guidelines actually advise for using the nightly one, which is what I did https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md
@ma2bd What's the preferred way here? Should we stick to nightly formatting or change the contributing guidelines?

Reply:
`cargo +nightly fmt` is enforced in CI. We don't pin the version of the nightly but we could if needed.